app/src/components/market/sections/market_buy/scalar_market_buy.tsx

```diff
@@ -371,7 +377,7 @@ export const ScalarMarketBuy = (props: Props) => {
             <TransactionDetailsLine />
             <TransactionDetailsRow
               title={'Max. Loss'}
-              value={`${!amount.isZero() ? '-' : ''}${formatNumber(formatBigNumber(amount, collateral.decimals))} ${
+              value={`${formatNumber(formatBigNumber(amount, collateral.decimals, collateral.decimals))} ${
```

Comment:
We should keep this `${!amount.isZero() ? '-' : ''}` part to make sure we include the negative sign here

Reply:
so the reason i removed this was to solve the problem that @Mi-Lan brought up.

do you want max loss to say you're losing negative 7 dollars? wouldn't that mean you're gaining 7 dollars because a 7 dollar debt is being removed from you? two negatives = positive. personally i think semantically this makes more sense and solves the above mentioned problem but if you think i should try to solve this differently i will definitely try!

> > ![Screenshot 2021-03-24 at 14 11 10](https://user-images.githubusercontent.com/33226956/112315847-d329e800-8caa-11eb-95a1-ca79df07a75b.png)
> > I think minus sign in the max los should be after less then sign...
> 
> Fixed in latest commit

Reply:
I agree it's a bit confusing to have a negative max. loss. Curious @pimato's take on this

Reply:
I think it is fine with having no `-` in the Max. Loss value